src/components/app/App.js

```diff
@@ -1,5 +1,19 @@
 import React from 'react';
+import { 
+  BrowserRouter as Router,
+  Route, 
+  Switch
+} from 'react-router-dom';
+import { TopWords, getFirstWords } from '../../containers/words/TopWords';
 
 export default function App() {
-  return <h1>Hi</h1>
+  return (
+    <Router>
+      <Switch>
+        <Route exact path="" component={TopWords} />
+        <Route exact path="/words/:count" component={TopWords} />
+        <Route exact path="/" component={getFirstWords} />
```

Comment:
```suggestion
        <Route exact path="/" component={TopWords} />
```
I would try getting your default path to display the top words before moving to the searchable portion.